File: app/src/main/java/com/example/topsedtest/customs/textViews/DigitalTextView.java
package com.example.topsedtest.customs.textViews;

import android.content.Context;
import android.graphics.Typeface;
import android.util.AttributeSet;

import androidx.appcompat.widget.AppCompatTextView;

public class DigitalTextView extends AppCompatTextView {

    public DigitalTextView(Context context, AttributeSet attrs) {
        super(context, attrs);

        String fontName = "fonts/digital.ttf";
        Typeface typeface = Typeface.createFromAsset(context.getAssets(), fontName);
        setTypeface(typeface);
    }
}

File: app/src/main/java/com/example/topsedtest/customs/textViews/DigitalTextView.kt
package com.example.topsedtest.customs.textViews

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatTextView

class DigitalTextView(context: Context, attrs: AttributeSet?) :
    AppCompatTextView(context, attrs) {
    init {
        val fontName = "fonts/digital.ttf"
        val typeface = Typeface.createFromAsset(context.assets, fontName)
        setTypeface(typeface)
    }
}